product_docs/docs/eprs/7/10_appendix/03_resolving_problems/04_troubleshooting_areas.mdx

```diff
@@ -188,7 +188,7 @@ Changed database context to 'edb'.
 7> GO
```

Comment:
Similar to a comment on another page, since SQL Server 2008 is also no longer supported anymore, the above reference to it and the corresponding commands should be removed.